### DISCUSSION TITLE
Merge to esphome/esphome?

### DISCUSSION THREAD

**Discussion by ludoviclehmann** [emporia-vue-local/esphome#87] [General]
Just wondering, is there any plan to merge this to esphome/esphome? 
I am running several emporia vue with this code and esphome + mqtt and did not encounter any issues until now :)

I saw that there was one pull request for emporia vue on esphome/esphome, but it has been cancelled a long time ago...

**Comment by flaviut** [comment]
Nope! They're already super busy, and they don't really have time to deal with fixing this project's bugs.

As far as level of effort goes, maintaining it here is really not any more work than maintaining it there, and it's very easy to use as an external component.

If someone else wants to submit it, please feel welcome to.
